tsconfig.json

```diff
@@ -10,7 +10,12 @@
 		"sourceMap": true,
 		"strict": true,
 		"strictNullChecks": true,
-		"moduleResolution": "bundler"
+		"moduleResolution": "bundler",
+		"baseUrl": ".",
```

Comment:
You have specified a baseUrl and/or paths in your tsconfig.json which interferes with SvelteKit's auto-generated tsconfig.json. Remove it to avoid problems with intellisense. For path aliases, use `kit.alias` instead: https://kit.svelte.dev/docs/configuration#alias